lib/api/actions.js

```diff
@@ -30,8 +30,8 @@ class Base {
   }
 
   response() {
-    if (this.object) {
-      if (this.object.toResponse) {
+    if (_.has(this, 'object')) {
```

Comment:
Let's just put `this.object = null` in the constructor, to have it always defined, no? These conditions do look not so good? What are the cases there?

Reply:
@alexkravets  We have the endpoint which returns number or null in the object. So if object is 0 or null this condition `if (this.object)` doesn't pass and the value which is returned by this endpoint is an empty array (`_.map(undefined)` returns empty array) which is unexpected behavior.
I have made these changes to have the possibility to return null, 0, false and undefined in the object if required.